Deps update, organize example, and add some docs. (#16)
mfeckie/phoenix_wings: README.md

chatChannel.join();
```

### Examples

*Mobile* - when running the flutter example in your emulator, with the server

> also running in the same computer host, remmember that the emulator is runing 

Reviewer: ```suggestion
also running in the same computer host, remember that the emulator is running 
```

in a segregated VM, so you need to configure it to point your server that is

> running in the host machine.

Reviewer: ```suggestion
running on the host machine.
```

```bash

# check your IP configuration

> sudo ifconfig

Reviewer: ```suggestion
$ ifconfig
```

enp0s20u5c4i2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
inet 10.0.0.2  netmask 255.255.255.240  broadcast 172.20.10.15
```

1. After checking your IP, go to your flutter Settings -> Proxy, and add the proxy

> host configuration with your IP, and port where you phoenix server with the

Reviewer: ```suggestion
host configuration with your IP, and port where your phoenix server with the
```

websockets is listening.

2. Configure your flutter app to point to your phoenix websocket server.
```
final socket = PhoenixSocket("ws://10.0.0.2:4000/socket/websocket");
```
See
[here](https://stackoverflow.com/questions/6760585/accessing-localhostport-from-android-emulator)

> for an ilustrated example.

Reviewer: ```suggestion
for an illustrated example.
```
